bookserver/routers/books.py

```diff
@@ -89,6 +107,11 @@ async def serve_page(
 ):
     rslogger.debug(f"user = {user}, course name = {course_name}")
     # Make sure this course exists, and look up its base course.
+    if user:
```

Comment:
Add a note somewhere that these values are being passed to JavaScript, hence `"true"` instead of `True` and so on (ick).